react/features/app/components/AbstractApp.js

```diff
@@ -251,6 +253,13 @@ export class AbstractApp extends BaseApp<Props, *> {
             }));
         this.nativeEventListeners.push(videoConfBridgeEmitter.addListener(NativeEvents.SHOW_SPEAKER_VIEW,
             (showSpeakerView: boolean | number) => dispatch(editSpeakerViewVisibility(Boolean(showSpeakerView)))));
+        this.nativeEventListeners.push(videoConfBridgeEmitter.addListener(NativeEvents.UPDATE_USER_AVATAR,
+            (jsonString: string) => {
+                const { xmppLoginId, info } = JSON.parse(jsonString);
```

Comment:
please do the json parse in the updateUserAvatar function instead of do in the lambda